\documentclass[journal=jacsat,manuscript=article]{achemso}
\usepackage[version=3]{mhchem}
\usepackage{amsmath}
\newcommand*\mycommand[1]{\texttt{\emph{#1}}}
\author{Brittany C. MacIntyre}
\author{Meera Shanmuganathan}
\author{Shannon L. Klingel}
\author{Zachary Kroezen}
\author{Erick Helmeczi}
\author{Na-Yung Seoh}
\author{Vanessa Martinez}
\author{Philip Britz-McKibbin}
\affiliation{Department of Chemistry and Chemical Biology, McMaster
University, Hamilton, ON L8S 3W3, Canada}
\email{britz@mcmaster.ca}
\author{David M. Mutch}
\affiliation{Department of Human Health and Nutritional Sciences,
University of Guelph, Guelph, ON N1G2W1,Canada}
\email{dmutch@uoguelph.ca}
\author{Adrian Chabowski}
\affiliation{Department of Physiology, Medical University of Bialystok,
15-222 Bialystok, Poland}
\author{Zeny Feng}
\affiliation{Department of Mathematics \& Statistics, University of
Guelph, Guelph, ON N1G 2W1, Canada}

\abbreviations{IR,NMR,UV}

\keywords{precision nutrition; metabolomics; omega-3 long-chain
polyunsaturated fatty acids (n3-LCPUFA); omega-3 index; dietary
biomarkers; urinary metabolites\LaTeX}

\title[An \textsf{achemso} demo]{Urinary metabolite profiling to
non-invasively monitor omega-3
index\footnote{Metabolites 2023, 13, 1071. https://doi.org/10.3390/metabo13101071}}
\makeatletter
\ifxetex
  \usepackage[setpagesize=false, % page size defined by xetex
              unicode=false, % unicode breaks when used with xetex
              xetex]{hyperref}
\else
  \usepackage[unicode=true]{hyperref}
\fi
\hypersetup{breaklinks=true,
            bookmarks=true,
            pdfauthor={},
            pdftitle={},
            colorlinks=true,
            urlcolor=blue,
            linkcolor=magenta,
            pdfborder={0 0 0}}
\urlstyle{same}  % don't use monospace font for urls


% tightlist command for lists without linebreak
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

% From pandoc table feature
\usepackage{longtable,booktabs,array}
\usepackage{calc} % for calculating minipage widths
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}

% Pandoc citation processing
%From Pandoc 3.1.8
% definitions for citeproc citations
\NewDocumentCommand\citeproctext{}{}
\NewDocumentCommand\citeproc{mm}{%
  \begingroup\def\citeproctext{#2}\cite{#1}\endgroup}
\makeatletter
 % allow citations to break across lines
 \let\@cite@ofmt\@firstofone
 % avoid brackets around text for \cite:
 \def\@biblabel#1{}
 \def\@cite#1#2{{#1\if@tempswa , #2\fi}}
\makeatother
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newenvironment{CSLReferences}[2] % #1 hanging-indent, #2 entry-spacing
 {\begin{list}{}{%
  \setlength{\itemindent}{0pt}
  \setlength{\leftmargin}{0pt}
  \setlength{\parsep}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1
   \setlength{\leftmargin}{\cslhangindent}
   \setlength{\itemindent}{-1\cslhangindent}
  \fi
  % set entry spacing
  \setlength{\itemsep}{#2\baselineskip}}}
 {\end{list}}
\usepackage{calc}
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}\break}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}


\begin{document}
\begin{abstract}
The Omega-3 Index (O3I) reflects eicosapentaenoic acid (EPA) and
docosahexaenoic acid (DHA) content in erythrocytes. While the O3I is
associated with numerous health outcomes, its widespread use is limited.
We investigated whether urinary metabolites could be used to non
invasively monitor the O3I in an exploratory analysis of a previous
placebo-controlled, parallel arm randomized clinical trial in males and
females (n = 88) who consumed either \textasciitilde3 g/d olive oil (OO;
control), EPA, or DHA for 12 weeks. Fasted blood and first-void urine
samples were collected at baseline and following supplementation, and
they were analyzed via gas chromatography and multisegment
injection--capillary electrophoresis--mass spectrometry (MSI-CE-MS),
respectively. We tentatively identified S-carboxypropylcysteamine (CPCA)
as a novel urinary biomarker reflecting O3I status, which increased
following both EPA and DHA (p \textless{} 0.001), but not OO
supplementation, and was positively correlated to the O3I (R = 0.30, p
\textless{} 0.001). Additionally, an unknown dianion increased following
DHA supplementation, but not EPA or OO. In ROC curve analyses, CPCA
outperformed all other urinary metabolites in distinguishing both
between OO and EPA or DHA supplementation groups (AUC
\textgreater80.0\%), whereas the unknown dianion performed best in
discriminating OO from DHAalone (AUC=93.6\%). Candidate urinary
biomarkers of the O3I were identified that lay the foundation for a
non-invasive assessment of omega-3 status.
\end{abstract}
\section{Introduction}\label{introduction}

The omega-3 long-chain polyunsaturated fatty acids (n3-LCPUFA)
eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA) have
recognized anti-inflammatory and triglyceride-lowering
properties\textsuperscript{2} and are associated with reductions in
coronary heart disease mortality\textsuperscript{3}, neuropsychiatric
disorders\textsuperscript{\textbf{EmergingRoleOmega?}}, and all-cause
mortality\textsuperscript{4}. EPA and DHA are primarily obtained through
the consumption of fatty fish and other marine foods, but small amounts
are also produced endogenously\textsuperscript{6}. Due to the numerous
health benefits associated with n3-LCPUFA, monitoring their levels in
the population is important to help prevent and/or mitigate health
risks.

The two most common methods to determine a person's n3-LCPUFA status are
through dietary assessment or through directly measuring their levels in
the blood; however, both methods have recognized limitations.
Self-reported assessments of food intake, such as food frequency
questionnaires and diet records, are subject to reporting
biases\textsuperscript{8} and are unable to account for individual
differences in n3-LCPUFA digestion, absorption, and metabolism that
ultimately affect their levels in the body. The current gold standard to
determine a person's n3-LCPUFA status is to directly measure EPA and DHA
levels in erythrocytes using gas chromatography--flame ionization
detection (GC-FID) or mass spectrometry\textsuperscript{9}. The sum of
EPA and DHA in erythrocytes is commonly known as the Omega-3 Index
(O3I). In contrast to fatty acids measured in serum, plasma, or whole
blood, the O3I is less sensitive to acute dietary changes and was
reported to reflect n3 LCPUFA membrane content in other tissues such as
heart, liver, muscle, and kidney\textsuperscript{11}. Consequently, the
O3I has been positioned as a modifiable and diet-sensitive biomarker
that can be used to estimate cardiovascular disease (CVD) mortality and
other health risks, where an O3I value of \textless4\% corresponds to
high risk and \textgreater8\% corresponds to low
risk\textsuperscript{12}. However, a major limitation with measuring the
O3I is the need for a blood sample, which presents a challenge in many
people due to needle phobia\textsuperscript{14}, as well as the lack of
a globally accepted standardized methodology for calculating the
O3I\textsuperscript{15}.

To overcome the challenges associated with dietary assessments, blood
sampling, and complicated sample workup procedures, researchers have
begun to explore the use of urinary metabolites to objectively and
non-invasively monitor an individual's dietary intake and/or nutrient
status\textsuperscript{18}. Several studies have proposed candidate
urinary biomarkers related to fish intake, including acetylcarnitine,
methylbutyrylcarnitine, propionylcarni tine, and
3-methylhistidine\textsuperscript{19}; however, these urinary
metabolites remain to be independently validated and may not be specific
to n3-LCPUFA. The O3I was recently reported to be inversely associated
with the urinary albumin--creatinine ratio\textsuperscript{20} and
8-hydroxy-2 deoxyguanosine (8-OHdG;\textsuperscript{21}), while other
studies have reported urinary metabolites that change following
n3-LCPUFA supplementation, such as trimethylamine-N-oxide
(TMAO)\textsuperscript{22} and
3-carboxy-4-methyl-5-propyl-2-furanpropanoic acid
(CMPF)\textsuperscript{23}. How ever, no study to date has investigated
if urinary metabolites could serve as a biomarker for the O3I. As such,
identifying and validating urinary metabolites that reflect an
individual's O3I could potentially lay the foundation for the
development of a non-invasive biomonitoring strategy for precision
nutrition.

The present study is an exploratory secondary analysis of a previously
reported randomized control trial in which the differential effects of
high-dose EPA and DHA supplementation on the O3I and blood pressure
regulation were investigated\textsuperscript{24}. The current analysis
aimed to identify and evaluate candidate urinary metabolites that can
serve as non-invasive biomarkers of the O3I in a cohort of healthy young
adults who consumed EPAor DHAsupplements for 12 weeks. We hypothesized
that a small number of urinary metabolites would be responsive and
specific to changes in n3-LCPUFA status such that they could perform
reliably as a non-invasive biomarker of the O3I. An untargeted data
workflow was used to characterize changes in the urine metabolome via
multisegment injection--capillary electrophoresis--mass spectrometry
(MSI-CE-MS) and identify novel biomarkers temporally associated with EPA
and/or DHA intake.

\section{Materials and Methods}\label{materials-and-methods}

\subsection{Participants and Study
Design}\label{participants-and-study-design}

Ninety healthy men and women aged 18--30 years were recruited from the
University of Guelph (Guelph, Ontario, Canada) between January 2017 and
June 2017. Exclusion criteria included smoking, history of
cardiovascular disease, chronic use of pharmacological medications
(except for oral contraceptives), \textgreater2 servings of fish or
shellfish (or other EPA/DHAfortified foods) per week, and fish oil
supplementation in the previous 3 months. All participants provided
written informed consent, and the study was approved by the Research
Ethics Board at the University of Guelph. The trial was registered at
clinicaltrials.gov (NCT03378232).

Eighty-eight participants completed the 12-week double-blind, parallel
arm, randomized trial in which the effects of \textasciitilde3 g/day
olive oil (OO; control), EPA, or DHA supplementation were compared.
Participants were randomized into one of 3 groups, (1) OO supplement,
(2) EPA supplement, and (3) DHA supplement, with block randomization
stratified by sex. Participants were instructed to maintain previous
dietary and exercise habits for the duration of the trial and one day
prior to blood and urine collection\textsuperscript{24}. Following an
overnight fast, blood and first-void urine samples were collected before
(base line) and after the 12-week supplementation period and stored at
80 C until analysis. Participant characteristics, including age, height,
and weight, were measured at baseline.

\subsection{Supplements}\label{supplements}

Olive oil and purified EPA and DHA (KD-PUR DHA700TG) oil supplements
were obtained from KD Pharma (Bexbach, Germany) and encapsulated using
InnovaGel in identical softgel capsules. Fatty acid (FA) purity (mean
SEM) was determined via gas chromatography and previously reported to be
75.7 0.1\% for oleic acid (18:1n-9) in the olive oil supplement, 74.7
0.09\% EPA and 0.55 0.01\% DHA in the EPA oil supplement, and 72.3 1.3\%
DHA and 1.05 0.11\% EPA in the DHA oil supplement\textsuperscript{25}.
Quantitative fatty acid purity levels were also reported
previously\textsuperscript{24}, with the OO capsules containing 818 mg
oleic acid and no detectable EPA or DHA, EPA capsules containing 813 mg
EPA and 7 mg DHA, and DHA capsules containing 814 mg DHA and 7.5 mg EPA.
Supplementation of 4 capsules/d at these amounts corresponded to
approximately 3 g/d of oleic acid, EPA, and DHA.Participants were
instructed to take 2 capsules twice daily with food for 12 weeks.

\subsection{Erythrocyte Fatty Acid
Content}\label{erythrocyte-fatty-acid-content}

Blood samples were collected from the antecubital vein into EDTA-treated
vacutainers to isolate erythrocytes. Samples were separated via
centrifugation at 700 g at 4 C for 15 min. Erythrocyte samples were
shipped to the Medical University of Białystok, Poland, on dry ice for
fatty acid composition analysis. Lipids were extracted according to the
Folch method\textsuperscript{26}. Briefly, lipids from isolated
erythrocytes were extracted in chloro form:methanol (2:1 vol:vol)
containing butylated hydroxy-toluene (0.01\%) as an antioxidant and
heptadecanoic acid (17:0) as an internal standard, as previously
described\textsuperscript{27}. After lipid extraction and
transmethylation with BF3/methanol, the lipid phase containing fatty
acid methyl esters (FAMEs) was dissolved in hexane and analyzed using a
Hewlett-Packard 5890 Series II gas chromatograph with a Varian CP-SIL
capillary column (100 m, internal diameter 0.25 mm) and flame-ionization
detector. Individual fatty acids were detected in accordance with the
retention times of standards. FAME and fatty acid standards were
purchased from Larodan. A total of 15 FAs were measured: myristic acid
(14:0), palmitic acid (16:0), palmitoleic acid (16:1n-7), stearic acid
(18:0), oleic acid (C18:1n-9), linoleic acid (18:2n-6), arachidic acid
(20:0),-linolenic acid (ALA; 18:3n-3), behenic acid (22:0),
di-homo--linolenic acid (20:3n-6), arachidonic acid (20:4n-6),
lignoceric acid (24:0), EPA (20:5n-3), nervonic acid (24:1n-9), and DHA
(22:6n-3). FAs were standardized to haemoglobin (Hb) and reported as
relative (\% FA composition) values or quantitative (ng FA/mg Hb)
values. The O3I was calculated through summing the relative \% of EPA
and DHA.

\subsection{Erythrocyte Fatty Acid
Content}\label{erythrocyte-fatty-acid-content-1}

\subsubsection{Chemicals}\label{chemicals}

All chemical standards and calibrants were purchased from Sigma-Aldrich
(St.~Louis, MO, USA), including analytical grade ammonium acetate,
ammonium bicarbonate, ammonium hydroxide, formic acid, organic acids,
sodium hydroxide, and recovery/internal standards, including
4-aminobutyric acid-2,2,3,3,4,4-d6 (GABA-d6), choline-d9,
4-fluorophenylalanine (F-Phe), 3-chlorotyrosine (Cl-Tyr),
4-fluorotyrosine (F-Tyr) and naphthalene-2 sulfonic acid (NMS). All
LC-MS grade solvents, including acetonitrile, isopropanol, methanol, and
water, were obtained from Caledon Laboratories Ltd.~(Georgetown, ON,
Canada). Calibrant solutions for all analytes were prepared through the
serial dilution of stock solutions (50 mM) in LC-MS grade water and
stored in a refrigerator (4 C).

\subsubsection{Preparation of Urine Samples and Quality
Controls}\label{preparation-of-urine-samples-and-quality-controls}

Briefly, all urine samples were slowly thawed on ice and centrifuged at
10,000 g for 5 min. Subsequently, 20 L of urine was aliquoted into a
centrifuge tube and diluted five-fold with 60 L deionized water, and a
20 L mixture of internal standard (200 M Cl-Tyr and NMS) and recovery
standards (200 M GABA-d6, choline-d9, F-Phe, F-Tyr), for a total volume
of 100 L. Centrifuge tubes were vortexed for 5 s; then, 20 L was
aliquoted into a polypropylene vial, and each sample was analyzed in
duplicate. Quality controls (QCs) were made to represent the average of
all combined samples, to be used to assess instrument drift from run to
run. Pooled QCs were prepared through aliquoting 2 L from each
individual sample into a centrifuge tube to establish a representative
average sample. Then, a 20 Laliquot was diluted five-fold with 40 L
deionized water, and 40 L internal and recovery standard mixture. This
QC sample was analyzed in every analytical run via MSI-CE-MS in
randomized sample injection positions during data acquisition for the
evaluation of technical precision for each urinary metabolite that
satisfied our selection criteria (CV \textless{} 30\% with detection
frequency \textgreater60\%) using a high-throughput MSI-CE-MS
metabolomics platform and standardized data workflow for biomarker
discovery\textsuperscript{28}. Three subgroups of QC samples were also
prepared through aliquoting separately from the control, EPA, and DHA
supplement arms, which were used for nontargeted metabolite profiling in
conjunction with a blank sample. All diluted urine samples were stored
at 80 Cprior to analysis.

\subsubsection{Urine Metabolome
Analysis}\label{urine-metabolome-analysis}

AnAgilent 6230B time-of-flight (TOF) mass spectrometer with an
electrospray ionization (ESI) source equipped to an Agilent G7100A
capillary electrophoresis (CE) unit was used for all experiments
(Agilent Technologies Inc., Mississauga, ON, Canada). An Agilent 1260
Infinity isocratic pump and a 1260 Infinity degasser were applied to
deliver sheath liquid. A sheath liquid composition of 0.1\% vol formic
acid in (60:40 methanol:water) and (70:30 methanol:water) at a flow rate
of 10 L/min were used for positive and negative ion mode, respectively.
For real-time mass correction, purine (20 L) and hexakis(2,2,3,3
tetrafluoropropoxy)phosphazine (HP-921, 20 L) reference ions were spiked
into the sheath liquid (400 L) at 0.02\% vol to provide constant mass
signals. The nebulizer spray was set off during the serial sample
injection before being switched on at 4 psi (27.6 kPa) following voltage
application. The source temperature was set to 300 C, and drying gas was
delivered at 4 L/min. The instrument was operated under a 2 GHz extended
dynamic range for positive and negative modes of detection. The Vcap,
fragmentor, skimmer, and octupole RF voltage were set to 3500 V, 120 V,
65 V, and 750 V, respectively. Separations were performed on bare
fused-silica capillaries with a 50 m internal diameter, a 360 m outer
diameter, and a total length of 135 cm (Polymicro Technologies Inc.,
Phoenix, AZ, USA). A capillary window maker (MicroSolv, Leland, NC, USA)
was used to remove 7 mm of the polyimide coating on both ends of the
capillary. All diluted urine and pooled QC samples were analyzed via
MSI-CE-MS under two different configurations with full data acquisition.
Sub-group analysis was also performed via MSI-CE-MS on pooled urine
samples from different sub-groups of participants. An acidic background
electrolyte (BGE, 1.0 M formic acid with 15\% vol acetonitrile, pH 1.8)
was used for resolving cationic metabolites under positive ion mode
while an alkaline BGE (50 mM ammonium bicarbonate, pH 8.5, adjusted with
ammoniumhydroxide) was used for anionic metabolites under negative ion
mode, as described elsewhere\textsuperscript{30}. Diluted urine samples
were introduced hydrodynamically at 100 mbar (10 kPa), alternating
between 5 s for each sample plug and 75 s for each BGEspacer plug, which
was electrokinetically injected at 30 kV. In total, thirteen discrete
samples were analyzed within a single analytical run of 45 min. The
applied voltage was set to 30 kV at 25 C for CE separations together
with a gradient pressure of 2 mbar/min. The structural elucidation of
urinary metabolites associated with the O3I was performed via
collision-induced dissociation experiments at an optimum collision
energy when using a single-injection format in CE coupled to a 6550
quadrupole--time of flight--mass spectrometer system (Agilent
Technologies Inc.) under positive or negative ion
modes\textsuperscript{31}. The structural elucidation of unknown
metabolites associated with the O3I was also supported with accurate
mass database searches for putative candidate ions reported in the Human
Metabolome Database (HMDB 5.0)\textsuperscript{32}, as well as predicted
MS/MS spectra generated in silico via CFM-ID 4.0\textsuperscript{33}.

\subsubsection{Comprehensive Analysis of Ionic Urinary Metabolites with
Quality
Control}\label{comprehensive-analysis-of-ionic-urinary-metabolites-with-quality-control}

A targeted and non-targeted data workflow was applied for characterizing
urinary metabolites from spurious signals and background ions when using
temporal signal pattern recognition with multiplexed separations in
MSI-CE-MS\textsuperscript{34}. An iterative approach was used to
authenticate urinary metabolites primarily as their intact molecular
ions (i.e., {[}M+H{]}+ or{[}M H{]} )while filtering out signal
redundancy, such as in-source fragments, isotopic contributions, and
adducts. Briefly, a sub-group analysis was first performed when using a
dilution trend filter comprising 13 independent serial sample injections
via MSI-CE-MS under positive and negative ion mode\textsuperscript{35},
including a quadruple injection of a pooled urine sample comprised of
all individuals in the study at both time points together with a single
blank sample. An open-source software for data pre-processing of
metabolomic data sets, MZmine (version 2.53)\textsuperscript{36}, was
used for reviewing, annotating, and selecting molecular features based
on their characteristic accurate mass (m/z) and relative migration time
(RMT) detected under positive (p) or negative (n) ion modes. Also, an
in-house urinary metabolite library was used for targeted analysis,
where each urinary metabolite was processed using vendor specific
software (Agilent MassHunter Qualitative Analysis, version 10.0). The
integration of all urinary molecular features was normalized to an
internal standard (Cl-Tyr or NMS under positive and negative ion mode,
respectively) and reported as their relative peak area (RPA). Temporal
signal pattern recognition when using multiplexed separations in
MSI-CE-MS was used to reject spurious, background, and redundant (i.e.,
in-source fragments, isotopic signals, salt adducts, and same ion
measured in both modes) molecular features when analyzing urine samples
pooled from OO, EPA, and DHA sub-groups. Overall, 125 urinary
metabolites satisfied the following inclusion criteria: (1) detected
with adequate frequency (\textgreater60\%) and technical precision
(coefficient of variance, CV \textless{} 30\%) after a repeat analysis
of 33 QC runs comprised of pooled urine from all participants, and (2)
detected with adequate frequency (\textgreater75\%) in participant data
measured at both time points (n = 176). This filtering approach resulted
in a urine metabolome data matrix that excluded infrequently detected
exogenous compounds (e.g., acetaminophen glucuronide, saccharin, etc.).
Additionally, recovery standards were excluded from the urine metabolome
data matrix (except F-Phe). Otherwise, urinary metabolites not detected
in certain samples were replaced with half the value of the minimum
response measured across the entire cohort if present below method
detection limits, whereas putative missing data due to potential
isobaric interferences were excluded from analysis. Lastly, the
integrated response ratio for each metabolite to an internal standard or
RPA was normalized to creatinine for each respective participant to
correct for hydration status when relying on spot morning urine samples.
Overall, urinary metabolites associated with the O3I were identified
with high confidence (level 1) when spiked with authentic standards (if
available), tentatively identified based on the annotation of MS/MS
spectra with a consistent electrophoretic mobility (level 2), or were
unidentified with an unknown chemical structure (level 3) if ion
responses were too low to acquire MS/MS spectra without a suitable
candidate reported in the HMDB. In all cases, urinary metabolites were
annotated based on their characteristic accurate mass and relative
migration time (m/z:RMT) under positive (p) or negative (n) ion mode
together with their most likely molecular formula.

\subsection{Statistical Analysis}\label{statistical-analysis}

All statistical analyses were conducted in R(v4.1.2)\textsuperscript{37}
with RStudio (v2022.7.2.576)\textsuperscript{38}. Rscripts are available
upon request.

\subsubsection{Multivariate Analysis}\label{multivariate-analysis}

Principal component analysis (PCA) was performed on the delta values of
FAs, the O3I, and urine metabolites to compare the effect of OO, EPA,
and DHA supplementation using the base-R stats function `princomp' and
`prcomp' for fatty acid and metabolite data, respectively. Scaled PCA
visualization plots were created with the package `factoextra' (v
1.0.7)\textsuperscript{39}. Partial least square--discriminant analysis
(PLS-DA) was performed on metabo lite variable delta values with the
`mixOmics' package (v 6.16.3)\textsuperscript{40}, where overall and
balanced error rates (BER) were plotted to determine the optimal number
of components for the final model. Sparse partial least
square--discriminant analysis (sPLS-DA) was used to select for
metabolites that responded to OO, EPA, or DHA supplementation.

\subsubsection{ANOVA Tests}\label{anova-tests}

Data normality and homogeneity of variance were assessed with the base-R
Shapiro Wilk test and Levene's test from the `rstatix' package (v
0.7.0.999)\textsuperscript{41}, respectively. For normal and non-normal
data distributions, one-way analysis of variance (ANOVA) or
Kruskal--Wallis tests were used to compare supplement groups at
baseline, respectively. To determine the effects of EPA and DHA
supplementation over time, creatinine-normalized urinary metabolite data
were log10-transformed to establish normal distribution, and a linear
mixed effects model (LMM) with random effects was fitted for each
participant (`nlme' R package, v 3.1.157)\textsuperscript{42} prior to
two-factor, repeated-measures ANOVA analysis (`stats' R package, v
4.1.2). For metabolites with a significant group time interaction effect
(Pint \textless{} 0.05), a Bonferroni control was applied to
within-factor post hoc pairwise t-tests using the `rstatix' and
`emmeans' packages (v 1.4.8)\textsuperscript{43}. For urinary
metabolites not normally distributed after log10 transformation, a
one-way Kruskal--Wallis test was performed at both time points followed
by post hoc pairwise comparisons using Dunn's test with Bonferroni
adjustment (`rstatix' package). For all post hoc pairwise comparisons,
p-adjusted values are reported. All analyses on erythrocyte FAs were
conducted on relative \%data; however, when possible, we also
investigated quantitative (ng FA/mg Hb) data to confirm our findings.

\subsubsection{Integrated Data}\label{integrated-data}

Erythrocyte FA and urine metabolome datasets were integrated in a single
dataset, with urinary metabolites as covariates and the O3I as the
response variable. Isobaric or highly colinear urinary metabolites were
removed prior to feature selection. Urinary metabolites in the
integrated dataset were scaled from 0 to 10 to match the range in the
O3I (the outcome variable). Feature selection was performed with a
regularized LMM to select for relevant metabolites while including a
random effect for participant. The LMM was implemented with the R
package `glmmLasso' (v 1.6.2)\textsuperscript{44} with an L1 penalty.
The tuning parameter lambda ( ) was selected through a grid search over
a given range of values from 2000 to 0.0001, where the value with the
lowest Bayesian information criterion (BIC) was selected for the final
fitted model. All remaining metabolites in the final model with non-zero
coefficients and p \textless{} 0.05 were identified. As a confirmation,
feature selection was also conducted on a second integrated dataset with
urine metabolites as covariates, but this time using quantitative EPA +
DHA (ng FA/mg Hb) as the response variable, which was scaled from 0--700
to match the EPA + DHA (ng FA/mg Hb) range in the dataset. The
diagnostic ability of selected creatinine-normalized urinary metabolites
and their ratios were assessed using receiver operating characteristic
(ROC) curves with the R package `pROC' (v 1.18.0)\textsuperscript{45},
where optimal thresholds were selected via optimality criterion
max(sensitivities + specificities), i.e., Youden's J statistic.

\section{Results and discussion}\label{results-and-discussion}

\subsection{Patient Characteristics}\label{patient-characteristics}

A total of 90 participants were enrolled into the study, where 30 (15
males and 15 females) were randomly allocated to each supplement group
(See Figure \ref{fig:fig1} . Two participants did not provide final
blood and urine samples, one in the EPA supplement group and one in the
OO control group, the former of which did not provide all baseline
samples/measurements. In total, data for 88 participants with matching
blood and urine samples at both time points were used for the present
analysis.

\begin{figure}
\centering
\includegraphics[width=3.125in,height=6.25in]{C:/Users/user1/Desktop/O3I_Study/Figures/Participants_study_design.png}
\caption{Flow diagram randomized controlled trial}\label{fig:fig1}
\end{figure}

\subsection{Baseline Participant Data Analysis in the Three Supplement
Groups}\label{baseline-participant-data-analysis-in-the-three-supplement-groups}

Group means for baseline participant data were calculated and are
reported in Table 1. There were no significant differences between the
three groups for baseline characteristics.

\begin{longtable}[]{@{}llll@{}}
\toprule\noalign{}
\textbf{Characteristics} & \textbf{OO} & \textbf{EPA} & \textbf{DHA} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
\emph{M/F,n} & 15/15 & 15/14 & 15/15 \\
\emph{Age,y} & 21.1 +/- 1.9 & 21.4 +/- 2.2 & 22.2 +/- 2.3 \\
\emph{Weight,kg} & 69.0 +/- 10.9 & 71.0 +/- 12.0 & 72.7 +/- 14.9 \\
\emph{BMI,kg/m2} & 24.1 +/- 3.5 & 23.1 +/- 2.8 & 23.7 +/- 3.4 \\
\end{longtable}

\subsection{Identification of Urinary Metabolites Associated with EPA
and/or DHA
Supplementation}\label{identification-of-urinary-metabolites-associated-with-epa-andor-dha-supplementation}

To avoid spurious results, we applied multiple statistical methods to
identify urinary metabolites associated with the O3I. These urinary
metabolites comprised 17 (out of 125) annotated unknown ions via
high-resolution MS tentatively identified using collision induced
dissociation MS/MS. First, we used supervised and unsupervised
clustering methods to obtain a general overview of the urine metabolome
data variance that distinguishes the supplement groups from one another.
Second, we used pairwise analyses to examine changes in
creatinine-normalized metabolite responses in relation to
supplementation and time. Finally, we investigated those urinary
metabolites that directly contributed to predicting variability in the
O3I. Collectively, these complimentary statistical approaches were used
to select and identify urinary metabolites that were consistently and
temporally associated with n3-LCPUFA status as compared to the OO
control.

\subsection{Distinguishing Supplementation Groups with Unsupervised and
Supervised Clustering
Approaches}\label{distinguishing-supplementation-groups-with-unsupervised-and-supervised-clustering-approaches}

First, we performed a global analysis to examine if urinary metabolite
profiles could distinguish between the three supplement groups from
baseline. Overall, the technical variance for repeated analysis of 125
metabolites (including creatinine) in a pooled urine sample was
acceptable with a median CV = 13.4\% for QCs (n = 33) as compared to
their greater between-subject biological variance with a median CV =
60.5\% (n = 178). Two-dimensional PCA plots of 124 creatinine-normalized
urinary metabolites at both baseline (Figure \ref{fig:fig2}) and 12
weeks were unable to distinguish the three supplement groups from one
another. Furthermore, a PCA plot corresponding to the delta response
values (i.e., RPA after supplementation -- RPA at baseline) for urinary
metabolites was also unable to distinguish the three supplement groups,
with the first two components (PC1 and PC2) only explaining 20.4\% of
the total variance in the dataset. Similarly, the three supplement
groups were not well distinguished by relative \%FA or by quantitative
FA data. In all instances, participants from all groups were clustered
around the centre of each plot with no clear distinction observed
between groups. We next performed supervised PLS-DA and sparse PLS-DA
(sPLS-DA) to better distinguish the three supplementation groups in this
study (Figure \ref{fig:fig2}). While less total variation was explained
by the first two principal components (i.e., PC1 + PC2 = 14\% and 9\%
for Figure 2a, b, respectively) than in the PCA model, the visual
separation between groups was slightly improved. Since we hypothesized
that only a small subset of urinary metabolites would relate to changes
in EPA and/or DHA erythrocyte content, we next performed a sPLS-DA for
urinary metabolite biomarker selection.

\begin{figure}
\centering
\includegraphics[width=6.25in,height=2.60417in]{C:/Users/user1/Desktop/O3I_Study/Figures/PC1_O3I.png}
\caption{Participant plots of relative peak area (RPA) delta values for
(a) 124 creatinine-normalized metabolite responses in PLS-DA and (b) 15
selected urinary metabolites for each principal component in sPLS-DA,
coloured by group (OO = orange, EPA = green, DHA =
blue).}\label{fig:fig2}
\end{figure}

Variable selection via sPLS-DA identified the top-ranked 15 urinary
metabolites associated with OO, EPA, and DHA supplementation for both
PC1 and PC2 (Figure \ref{fig:fig2}). The metabolites selected in PC1
were related to DHA supplementation (blue), whereas those selected in
PC2 largely corresponded to metabolites related to EPA supplementation
(green). The top-ranked urinary metabolites having the largest weighting
on PC1 were an unknown divalent anion (221.075:0.927:n), as well as an
unknown cation (164.074:0.614:p) that was tentatively identified as
S-carboxypropylcysteamine (CPCA) based on annotation of their slower
positive mobility (at pH 1.8) due to a more acidic alpha-carboxylic acid
moiety, resulting in their longer apparent migration times (e.g.,
S-propylcysteine). Overall, urinary CPCA was measured consistently with
acceptable technical precision (CV = 9.8\%, n = 33) throughout the
study. The third and fourth most significant urinary metabolites
classified via sPLS-DA along PC1 were tetrahydroaldosterone glucuronide
and an unknown anion isomer (241.120:0.653:n), which was tentatively
identified as a dipeptide when using MS/MS in negative ion mode, namely
pyroglutamylisoleucine (pGlu-Ile) (Figure \ref{fig:fig4}). As there were
two partially resolved isobars in the extracted ion electropherogram
with analogous MS/MS spectra, isobars were distinguished by their
apparent migration times, with the slower migrating isomer likely being
pyroglutamylleucine (pGlu-Leu). Overall, the urinary metabolites
contributing the most loading weight for PC1 (Figure S2a) were an
unknown dianion (221.075:0.927:n), CPCA (164.074:0.613:p),
tetrahydroaldosterone glucuronide, and pGlu-Ile. Top loading weights for
PC2 (Figure S2b) corresponded to choline, glucuronic acid, an unknown
dianion (88.004:1.622:n), and quinic acid. Structural elucidation of the
unknown dianion (221.075:0.927:n) was not feasible due to its low
abundance in urine that prevented the acquisition of an MS/MS spectra.
Its MS/MS spectrum at an optimal collisional energy under positive ion
mode (Figure \ref{fig:fig3}) together with its co-migration after
spiking a standard of CPCA in pooled urine. Other isobaric candidate
ions having the same molecular formula (e.g., ethione,
N-methylmethionine, S-methylmethionine, etc.) were excluded as likely
candidates based.

\begin{figure}
\centering
\includegraphics[width=6.25in,height=4.16667in]{C:/Users/user1/Desktop/O3I_Study/Figures/Chromatograms.png}
\caption{Structural elucidation of S-carboxypropylcysteamine
(CPCA).}\label{fig:fig3}
\end{figure}

\subsection{Increases in Urinary CPCA Are Associated with n3-LCPUFA
Erythrocyte
Content}\label{increases-in-urinary-cpca-are-associated-with-n3-lcpufa-erythrocyte-content}

Finally, we performed additional analyses on the two lead urinary
biomarkers of the O3I (CPCA and the unknown dianion 221.075:0.927:n)
that were systematically identified in all three statistical approaches.
Specifically, we examined whether the changes observed in these urinary
metabolites were specific to changes in the O3I through measuring
Pearson correlation coefficients ® with all other measured erythrocyte
fatty acids (Figure \ref{fig:fig4}). Urinary CPCA was positively
correlated with the O3I (r = 0.30, p \textless{} 0.001) but also showed
correlations with both EPA (r = 0.19, p = 0.012) and DHA (r = 0.21, p =
0.004) individually. CPCA was also inversely correlated with AA (r =
0.30, p \textless{} 0.001) and myristic acid (r = 0.26, p \textless{}
0.001). However, CPCA was not correlated with any other measured fatty
acid. The unknown dianion (221.075:0.927:n) was more strongly correlated
with the O3I (r = 0.47, p \textless{} 0.001) and with DHA (r = 0.53, p
\textless{} 0.001) but was not correlated with EPA.

\begin{figure}
\centering
\includegraphics[width=5.20833in,height=5.20833in]{C:/Users/user1/Desktop/O3I_Study/Figures/Most_relevant_metabolites.png}
\caption{Correlation plot of relative \% fatty acids in erythrocytes and
top 17 selected metabolites, where significant (p \textless{} 0.05 below
diagonal, Padj \textless{} 0.05 above diagonal) correlations are
indicated by sphere size (strength) and colour (blue = positive, red =
negative). \label{fig4}}\label{fig:fig4}
\end{figure}

\section{Discussion}\label{discussion}

To the best of our knowledge, this study is the first to investigate
urinary metabolites as putative non-invasive biomarkers of the O3I using
a non-targeted metabolomics ap proach. Our results identified urinary
CPCA and an unknown dianion (221.075:0.927:n) as novel candidate urinary
biomarkers for O3I status in healthy, young individuals. While the
unknown dianion (221.075:0.927:n) was also identified as an indicator of
DHA sup plementation specifically, CPCA was found to be associated with
both EPA and DHA supplementation as compared to O\textsuperscript{46}.
More specifically, both candidate urinary metabo lites were positively
associated with the O3I, and when compared to all 15 measured
erythrocyte membrane-derived phospholipid fatty acids,
creatinine-normalized urinary CPCAlevels showed specificity in their
association with the O3I, whereas the unknown dianion (221.075:0.927:n)
showed a greater association with DHA\textsuperscript{47}. As a
supplement group classifier between EPA or DHA relative to OO, urinary
CPCA consistently outperformed other selected urinary metabolites when
using ROC curves (AUC \textgreater{} 80\%), whereas the unknowndianion
showed the highest performance specifically for DHA (AUC \textgreater{}
90\%) with poor sensitivity for EPA. Similarly, both CPCA (AUC = 73\%)
and the unknown dianion (AUC=89\%)outperformed other urinary metabolites
when classifying a low (\textless4\%) versus high (\textgreater8\%)
O3I\textsuperscript{48}. Taken together, our robust analyses have
positioned urinary CPCA and the unknown dianion (221.075:0.927:n) as
novel surrogates for the O3I. Collectively, this work lays the
foundation for developing a convenient and non-invasive approach to
assess an individual's n3-LCPUFA status\textsuperscript{49}.

Urinary biomarkers represent an attractive option for nutrient
monitoring. Traditional dietary assessments estimate nutrient status,
but are limited in their ability to accurately capture variation in food
quality, intake, absorption, and metabolism\textsuperscript{50}.
Therefore, biomarkers have the advantage of more accurately indicating
the amount of a nutrient available to tissues and organs. Current
validated nutrient biomarkers of n3-LCPUFA status such as the
O3I\textsuperscript{51} require a small blood sample. Recently, Ly et
al.\textsuperscript{52} reported two phosphatidylcholine species can
serve as circulating surrogate biomarkers of the O3I directly in serum
or plasma rather than hydrolyzed EPA and DHA measured from the
phospholipid fraction of erythrocyte membranes. Yet, access to blood
specimens may still represent a barrier to their use at both the
individual and population levels. In contrast, the use of urinary
metabolites as a proxy for the O3I could improve ease of monitoring
while capturing physiologically relevant information. Although numerous
reports in the literature have identified potential candidate biomarkers
of n3-LCPUFA status, very few have been validated and are therefore
currently of limited use. While a validation pipeline for biomarkers of
food intake and/or nutrient status has been
proposed\textsuperscript{53}, meeting all criteria for establishing the
biological validity and analytical performance of a biomarker remains a
challenge. Nevertheless, the power of a validated urinary biomarker for
nutrient status has wide-scale implications for use in personalized
health, nutrition and epidemiological research, and clinical risk
assessment for cardiovascular events and hypertriglyceridemia, amongst
others.

Through our analyses, we found that urinary CPCA was significantly
correlated with changes in the O3I and AA satisfying both a dose and
temporal response as criteria. Moreover, urinary CPCA was reliably
measured with acceptable technical precision (mean CV \textless{} 10\%)
with no missing values using a validated multiplexed separation platform
and data workflow for metabolomics based on MSI-CE-MS. Also, urinary
CPCA increased consistently following both EPA and DHA supplementation
in our cohort of young Canadian adults having a low average O3I status
at baseline. Overall, no other putative urinary biomarker associated
with the O3I identified in our study had higher sensitivity (89.6\% and
83.3\%) and specificity (72.4\% and 72.4\%) for both EPA and DHA
supplementation, respectively. We hypothesized that the inverse
relationship between CPCA and AA stems primarily from the known
reduction in AA membrane content observed with increased n3-LCPUFA
intake. Indeed, we found that when the relative levels of erythrocyte
EPA and DHA increased, there were proportional reductions in AA. In a
dose--response randomized controlled trial examining EPA + DHA
supplementation in healthy male and female participants, Flock et
al.\textsuperscript{54} reported that erythrocyte EPA and DHAcontent
increased concomitant with comparable reductions in AA content.
Similarly, Vidgren et al.\textsuperscript{54} showed significant
reductions in erythrocyte AA content in individuals
consumingafishoilsupplementoreatingafishdiet.
Ourfindingsalignwiththeseprevious studies. Due to the close relationship
between n3-LCPUFA and AA erythrocyte membrane content, the specificity
of CPCA related to the O3I may falsely implicate the latter. Indeed,
there was an equivalent strength of association of urinary CPCA
inversely with AA and positively with the O3I (r = 0.30) as compared to
EPA or DHAalone. Nevertheless, increases in the O3I concomitant with
decreases in AA are generally indicative of good health {[}56{]}.

Urinary metabolites related to fish intake have been proposed as
potential urinary biomarkers in previous studies. In a cross-sectional
analysis of the INTERMAP study, Gibson et al.\textsuperscript{2} found
TMAO and taurine were capable of distinguishing between low and high
fish intake in a Japanese subpopulation. Our study identified and
quantified urinary TMAO, but we did not find that this metabolite was
capable of distinguishing betweenlowandhighO3Ilevels. Takentogether,
thissuggests that TMAOmaybeaurinary biomarker related to marine meat
protein intake rather than n3-LCPUFA. A randomized controlled trial in
individuals with type-2 diabetes supplemented with fish oil, flaxseed
oil, or corn oil showed significant increases in urinary CMPF in the
fish oil group, which was also highly correlated to serum
CMPF\textsuperscript{3}. The association of urinary CMPF could not be
evaluated in our study as it was not detected via MSI-CE-MS. Therefore,
both TMAO and CMPFrequire further evaluation to clarify their use as
biomarkers of fish/fish oil intake versus n3-LCPUFA status.

Currently, there are nine isobaric candidate metabolites in the HMDB
that match the most likely molecular formula for the urinary metabolite
we have tentatively identified as CPCA(164.074:0.613:p-C6H13NO2S). While
the unambiguous identification of CPCA is ongoing, we have ruled out
four isobaric ions based on migration behaviour and structural
properties that impact their ionization state (i.e., pKa) and
corresponding migration times. Of the remaining isobaric ion candidates
reported in the HMDB, the structural properties of CPCA's functional
groups best reflect the migration time and characterization shown in the
extracted ion electropherogram (Figure 3). Furthermore, additional
studies are necessary to clarify the underlying biochemical link(s)
between n3-LCPUFA and CPCA as well as establish reference intervals for
urinary CPCA in larger populations to define optimal cut-off
concentrations associated with the O3I.

Additionally, while demonstrating astrongassociation to theO3I,
anunknowndianion (221.075:0.927:n) showed specificity in its
relationship to DHA supplementation with seemingly little-to-no
association with EPA supplementation. In the absence of a confirmed
identification for this urinary metabolite, it is challenging to
speculate on the biochemical significance of this finding and its link
with DHA; however, this currently unidentified metabolite may prove to
be a specific marker of DHA in the body and thus warrants continued
investigation. Improved concentration sensitivity is needed to measure
this lower abundance hydrophilic urinary metabolite, which is also
needed when acquiring better quality MS/MS spectra.

Our study has several strengths. First, the participant samples used
were collected during a randomized placebo-controlled trial that
included matching baseline and 12-week fasted blood and first-void urine
samples. Second, erythrocyte fatty acid measurements were examined using
both relative \% and quantitative (ng FA/mg Hb) values, thus al lowing
for the robust verification of the relationship between erythrocyte EPA
and DHA with CPCA. All three statistical methods performed on all 124
creatinine-normalized urinary metabolites confirmed similar results,
where CPCA and an unknown dianion (221.075:0.927:n) were consistently
identified using several different approaches. Addi tionally, we
conducted an initial proof-of-principle study 2 years prior to the
present full study that first identified a relationship between CPCA and
high-dose DHA intake relative to OOwith goodmutual agreement and
reproducibility, thus demonstrating the stability of CPCAinfrozen urine
samples following repeat analysis after thawing and long-term storage.
Finally, to rule out potential metabolite contamination, we analyzed
both lipid and aqueous extracts of the OO, EPA, and DHA capsules and did
not detect CPCA, the unknown dianion (221.075:0.927:n), and other
polar/hydrophilic biomarker candidates reported in this work,
reinforcing that the changes in concentration of these endogenous
urinary metabolites reflect biochemical changes in study participants
following high-dose EPAand/or DHAintake.

There are also several limitations to our study. First, we acknowledge
the small and relatively homogenous sample size of the current study.
Indeed, participants were all young (18--30 y) and healthy individuals
that do not necessarily reflect variation seen in free-living
populations. Thus, the generalizability of urinary CPCA and the unknown
dianion (221.075:0.927:n) as non-invasive biomarkers for the O3I needs
to be confirmed in populations that vary in age, ethnicity, genetics,
and health status. Additionally, other lifestyle factors were not
accounted for during the clinical trial, such as physical activity,
alcohol consumption, and dietary patterns. Future studies will need to
measure macro- and micronutrient intakes to ensure that changes in
urinary metabolites are specific to changes in n3-LCPUFA intake.
Furthermore, first-void urine samples used in our metabolomics analyses
are more susceptible to acute or diurnal changes in urine composition,
which maydiminish the accuracy of associations drawn between urine
metabolites and long-term changes in erythrocyte fatty acid composition.
As such, 24 h urine samples and/or several repeat spot urine samples
collected over time from the same individual could provide a more
reliable metabolite assessment of O3I status and improve the strength of
associations between urinary metabolites and erythrocyte fatty acid
composition. Future work needs to explore the stability of these
metabolites in urine through comparing fasting first-void urine samples
with 24 h samples, as well as day-to-day variability. Moreover, urine
samples were analyzed on a single metabolomic platform (MSI-CE-MS),
which is well suited for identifying polar or ionic compounds since they
are predominantly excreted in human urine. However, due to the
selectivity and sensitivity of liquid chromatography, gas
chromatography, and CE-MS techniques to capture different types of
compounds, the use of complementary instrumental platforms may uncover
additional urinary biomarkers of the O3I\textsuperscript{53}.
Ultimately, this could lead to the identification of a larger panel of
urinary metabolites that may reflect changes in the O3I compared to CPCA
and/or the unknown dianion (221.075:0.927:n) alone while also
independently validating biomarker outcomes across
platforms\textsuperscript{3}. Furthermore, it is unclear what minimum
dosage of n3 LCPUFA is needed to elicit a significant increase in these
candidate urinary biomarkers above endogenous background levels in human
urine. Since participants in the present trial received moderately high
doses of EPA or DHA (\textasciitilde3 g per day), future studies should
consider different doses and formulations of n3-LCPUFA supplementation.
Finally, the underlying biochemical connection between these candidate
urinary biomarkers and n3 LCPUFA remains to be elucidated.

\section{Conclusions}\label{conclusions}

We report the identification of a novel panel of urinary metabolites as
robust, non invasive biomarkers of n3-LCPUFA status . Specifically, our
findings reveal that urinary CPCAandanunknowndianion (221.075:0.927:n)
may serve as promising biomarkers for the O3I, and thereby lay the
groundwork for establishing an objective and non-invasive test for the
personalized assessment of n3-LCPUFA status. We anticipate that an
eventual non-invasive test could be used for early screening of health
risks and self-monitoring of the O3I following personalized dietary
changes. However, additional studies confirming the sensitivity and
specificity of these urinary metabolites as biomarkers of the O3I
require further validation prior to considering their clinical utility.

\section*{References}\label{references}
\addcontentsline{toc}{section}{References}

\phantomsection\label{refs}
\begin{CSLReferences}{0}{0}
\bibitem[\citeproctext]{ref-calderOmega3FattyAcids2010}
\CSLLeftMargin{(1) }%
\CSLRightInline{\href{https://doi.org/10.3390/nu2030355}{Calder, P. C.
\emph{Nutrients} \textbf{2010}, \emph{2} (3), 355--374}.}

\bibitem[\citeproctext]{ref-mozaffarianOmega3FattyAcids2011}
\CSLLeftMargin{(2) }%
\CSLRightInline{\href{https://doi.org/10.1016/j.jacc.2011.06.063}{Mozaffarian,
D.; Wu, J. H. Y. \emph{Journal of the American College of Cardiology}
\textbf{2011}, \emph{58} (20), 2047--2067}.}

\bibitem[\citeproctext]{ref-abdelhamidOmega3FattyAcids2018}
\CSLLeftMargin{(3) }%
\CSLRightInline{\href{https://doi.org/10.1002/14651858.CD003177.pub4}{Abdelhamid,
A. S.; Brown, T. J.; Brainard, J. S.; Biswas, P.; Thorpe, G. C.; Moore,
H. J.; Deane, K. H.; AlAbdulghafoor, F. K.; Summerbell, C. D.;
Worthington, H. V.; Song, F.; Hooper, L. \emph{Cochrane Database of
Systematic Reviews} \textbf{2018}, No. 11}.}

\bibitem[\citeproctext]{ref-harrisBloodN3Fatty2021}
\CSLLeftMargin{(4) }%
\CSLRightInline{\href{https://doi.org/10.1038/s41467-021-22370-2}{Harris,
W. S.; Tintle, N. L.; Imamura, F.; Qian, F.; Korat, A. V. A.; Marklund,
M.; Djoussé, L.; Bassett, J. K.; Carmichael, P. H.; Chen, Y. Y.;
Hirakawa, Y.; Küpers, L. K.; Laguzzi, F.; Lankinen, M.; Murphy, R. A.;
Samieri, C.; Senn, M. K.; Shi, P.; Virtanen, J. K.; Brouwer, I. A.;
Chien, K. L.; Eiriksdottir, G.; Forouhi, N. G.; Geleijnse, J. M.; Giles,
G. G.; Gudnason, V.; Helmer, C.; Hodge, A.; Jackson, R.; Khaw, K. T.;
Laakso, M.; Lai, H.; Laurin, D.; Leander, K.; Lindsay, J.; Micha, R.;
Mursu, J.; Ninomiya, T.; Post, W.; Psaty, B. M.; Risérus, U.; Robinson,
J. G.; Shadyab, A. H.; Snetselaar, L.; Sala-Vila, A.; Sun, Y.; Steffen,
L. M.; Tsai, M. Y.; Wareham, N. J.; Wood, A. C. \emph{Nature
Communications} \textbf{2021}, \emph{12} (1)}.}

\bibitem[\citeproctext]{ref-burdgeConversionAlinolenicAcid2002}
\CSLLeftMargin{(5) }%
\CSLRightInline{\href{https://doi.org/10.1079/BJN2002689}{Burdge, G. C.;
Wootton, S. A. \emph{British Journal of Nutrition} \textbf{2002},
\emph{88} (4), 411--420}.}

\bibitem[\citeproctext]{ref-plourdeExtremelyLimitedSynthesis2007}
\CSLLeftMargin{(6) }%
\CSLRightInline{\href{https://doi.org/10.1139/H07-034}{Plourde, M.;
Cunnane, S. C. \emph{Applied Physiology, Nutrition, and Metabolism}
\textbf{2007}, \emph{32} (4), 619--634}.}

\bibitem[\citeproctext]{ref-BiasDietaryreportInstruments}
\CSLLeftMargin{(7) }%
\CSLRightInline{\href{https://www.cambridge.org/core/journals/public-health-nutrition/article/bias-in-dietaryreport-instruments-and-its-implications-for-nutritional-epidemiology/F74F7C0AF47FDD40061A3684DA658731}{Bias
in dietary-report instruments and its implications for nutritional
epidemiology {\textbar} {Public} {Health} {Nutrition} {\textbar}
{Cambridge} {Core}}.}

\bibitem[\citeproctext]{ref-poslusnaMisreportingEnergyMicronutrient2009}
\CSLLeftMargin{(8) }%
\CSLRightInline{\href{https://doi.org/10.1017/S0007114509990602}{Poslusna,
K.; Ruprich, J.; Vries, J. H. M. de; Jakubikova, M.; Veer, P. van't.
\emph{British Journal of Nutrition} \textbf{2009}, \emph{101} (S2),
S73--S85}.}

\bibitem[\citeproctext]{ref-Omega3IndexCardiovascular}
\CSLLeftMargin{(9) }%
\CSLRightInline{\href{https://www.mdpi.com/2072-6643/6/2/799?uid=563b4f0a2a}{Omega-3
{Index} and {Cardiovascular} {Health}}.}

\bibitem[\citeproctext]{ref-ComprehensiveReviewChemistry}
\CSLLeftMargin{(10) }%
\CSLRightInline{\href{https://www.mdpi.com/2072-6643/10/11/1662}{A
{Comprehensive} {Review} of {Chemistry}, {Sources} and {Bioavailability}
of {Omega}-3 {Fatty} {Acids}}.}

\bibitem[\citeproctext]{ref-tuCorrelationsBloodTissue2013}
\CSLLeftMargin{(11) }%
\CSLRightInline{\href{https://doi.org/10.1016/j.plefa.2012.04.005}{Tu,
W. C.; Mühlhäusler, B. S.; Yelland, L. N.; Gibson, R. A.
\emph{Prostaglandins, Leukotrienes and Essential Fatty Acids}
\textbf{2013}, \emph{88} (1), 53--60}.}

\bibitem[\citeproctext]{ref-harrisOmega3IndexNew2004}
\CSLLeftMargin{(12) }%
\CSLRightInline{\href{https://doi.org/10.1016/j.ypmed.2004.02.030}{Harris,
W. S.; Schacky, C. von. \emph{Preventive Medicine} \textbf{2004},
\emph{39} (1), 212--220}.}

\bibitem[\citeproctext]{ref-FearNeedlesSystematic}
\CSLLeftMargin{(13) }%
\CSLRightInline{\href{https://onlinelibrary.wiley.com/doi/full/10.1111/jan.13818}{The
fear of needles: {A} systematic review and meta‐analysis - {McLenon} -
2019 - {Journal} of {Advanced} {Nursing} - {Wiley} {Online} {Library}}.}

\bibitem[\citeproctext]{ref-mcmurtryExposurebasedInterventionsManagement2016}
\CSLLeftMargin{(14) }%
\CSLRightInline{\href{https://doi.org/10.1080/16506073.2016.1157204}{McMurtry,
C. M.; Taddio, A.; Noel, M.; Antony, M. M.; Chambers, C. T.; Asmundson,
G. J. G.; Pillai Riddell, R.; Shah, V.; MacDonald, N. E.; Rogers, J.;
Bucci, L. M.; Mousmanis, P.; Lang, E.; Halperin, S.; Bowles, S.;
Halpert, C.; Ipp, M.; Rieder, M. J.; Robson, K.; Uleryk, E.; Votta
Bleeker, E.; Dubey, V.; Hanrahan, A.; Lockett, D.; Scott, J.
\emph{Cognitive Behaviour Therapy} \textbf{2016}, \emph{45} (3),
217--235}.}

\bibitem[\citeproctext]{ref-dempseyInfluenceDietarySupplemental2023}
\CSLLeftMargin{(15) }%
\CSLRightInline{\href{https://doi.org/10.3389/fnut.2023.1072653}{Dempsey,
M.; Rockwell, M. S.; Wentz, L. M. \emph{Frontiers in Nutrition}
\textbf{2023}, \emph{10}}.}

\bibitem[\citeproctext]{ref-rafiqNutritionalMetabolomicsClassification2021}
\CSLLeftMargin{(16) }%
\CSLRightInline{\href{https://doi.org/10.1093/advances/nmab054}{Rafiq,
T.; Azab, S. M.; Teo, K. K.; Thabane, L.; Anand, S. S.; Morrison, K. M.;
Souza, R. J. de; Britz-McKibbin, P. \emph{Advances in Nutrition}
\textbf{2021}, \emph{12} (6), 2333--2357}.}

\bibitem[\citeproctext]{ref-RoleMetabolomicsDetermination}
\CSLLeftMargin{(17) }%
\CSLRightInline{\href{https://www.cambridge.org/core/journals/proceedings-of-the-nutrition-society/article/role-of-metabolomics-in-determination-of-new-dietary-biomarkers/E267A2C47DECFB3B863C29760F85F1E1\#}{The
role of metabolomics in determination of new dietary biomarkers
{\textbar} {Proceedings} of the {Nutrition} {Society} {\textbar}
{Cambridge} {Core}}.}

\bibitem[\citeproctext]{ref-SchemeFlexibleClassification}
\CSLLeftMargin{(18) }%
\CSLRightInline{\href{https://link.springer.com/article/10.1186/s12263-017-0587-x}{A
scheme for a flexible classification of dietary and health biomarkers
{\textbar} {Genes} \& {Nutrition}}.}

\bibitem[\citeproctext]{ref-UrinaryBiomarkersDietary}
\CSLLeftMargin{(19) }%
\CSLRightInline{\href{https://academic.oup.com/nutritionreviews/article/78/5/364/5610624}{Urinary
biomarkers of dietary intake: A review {\textbar} {Nutrition} {Reviews}
{\textbar} {Oxford} {Academic}}.}

\bibitem[\citeproctext]{ref-filipovicBloodOmega3Fatty2021}
\CSLLeftMargin{(20) }%
\CSLRightInline{\href{https://doi.org/10.3389/fcvm.2021.622619}{Filipovic,
M. G.; Reiner, M. F.; Rittirsch, S.; Irincheeva, I.; Aeschbacher, S.;
Grossmann, K.; Risch, M.; Risch, L.; Limacher, A.; Conen, D.; Beer, J.
H. \emph{Frontiers in Cardiovascular Medicine} \textbf{2021},
\emph{8}}.}

\bibitem[\citeproctext]{ref-bigorniaOmega3IndexInversely2016}
\CSLLeftMargin{(21) }%
\CSLRightInline{\href{https://doi.org/10.3945/jn.115.222562}{Bigornia,
S. J.; Harris, W. S.; Falcón, L. M.; Ordovás, J. M.; Lai, C.-Q.; Tucker,
K. L. \emph{The Journal of Nutrition} \textbf{2016}, \emph{146} (4),
758--766}.}

\bibitem[\citeproctext]{ref-gibsonAssociationFishConsumption2020}
\CSLLeftMargin{(22) }%
\CSLRightInline{\href{https://doi.org/10.1093/ajcn/nqz293}{Gibson, R.;
Lau, C.-H. E.; Loo, R. L.; Ebbels, T. M.; Chekmeneva, E.; Dyer, A. R.;
Miura, K.; Ueshima, H.; Zhao, L.; Daviglus, M. L.; Stamler, J.; Van
Horn, L.; Elliott, P.; Holmes, E.; Chan, Q. \emph{The American Journal
of Clinical Nutrition} \textbf{2020}, \emph{111} (2), 280--290}.}

\bibitem[\citeproctext]{ref-leeDocosahexaenoicAcidReduces2019}
\CSLLeftMargin{(23) }%
\CSLRightInline{\href{https://doi.org/10.1152/ajpheart.00677.2018}{Lee,
J. B.; Notay, K.; Klingel, S. L.; Chabowski, A.; Mutch, D. M.; Millar,
P. J. \emph{American Journal of Physiology-Heart and Circulatory
Physiology} \textbf{2019}, \emph{316} (4), H873--H881}.}

\bibitem[\citeproctext]{ref-metherelCompoundspecificIsotopeAnalysis2019}
\CSLLeftMargin{(24) }%
\CSLRightInline{\href{https://doi.org/10.1093/ajcn/nqz097}{Metherel, A.
H.; Irfan, M.; Klingel, S. L.; Mutch, D. M.; Bazinet, R. P. \emph{The
American Journal of Clinical Nutrition} \textbf{2019}, \emph{110} (4),
823--831}.}

\bibitem[\citeproctext]{ref-folchSIMPLEMETHODISOLATION1957}
\CSLLeftMargin{(25) }%
\CSLRightInline{\href{https://doi.org/10.1016/S0021-9258(18)64849-5}{Folch,
J.; Lees, M.; Stanley, G. H. S. \emph{Journal of Biological Chemistry}
\textbf{1957}, \emph{226} (1), 497--509}.}

\bibitem[\citeproctext]{ref-Omega3FattyAcid}
\CSLLeftMargin{(26) }%
\CSLRightInline{\href{https://faseb.onlinelibrary.wiley.com/doi/full/10.1096/fj.201801857RRR}{Omega‐3
fatty acid supplementation attenuates skeletal muscle disuse atrophy
during two weeks of unilateral leg immobilization in healthy young women
- {Mcglory} - 2019 - {The} {FASEB} {Journal} - {Wiley} {Online}
{Library}}.}

\bibitem[\citeproctext]{ref-MetabolomicsRevealsElevated}
\CSLLeftMargin{(27) }%
\CSLRightInline{\href{https://link.springer.com/article/10.1007/s11306-019-1543-0}{Metabolomics
reveals elevated urinary excretion of collagen degradation and
epithelial cell turnover products in irritable bowel syndrome patients
{\textbar} {Metabolomics}}.}

\bibitem[\citeproctext]{ref-yamamotoMetabolomicsRevealsElevated2019}
\CSLLeftMargin{(28) }%
\CSLRightInline{\href{https://doi.org/10.1007/s11306-019-1543-0}{Yamamoto,
M.; Pinto-Sanchez, M. I.; Bercik, P.; Britz-McKibbin, P.
\emph{Metabolomics} \textbf{2019}, \emph{15} (6), 82}.}

\bibitem[\citeproctext]{ref-yamamotoUrinaryMetabolitesEnable2021}
\CSLLeftMargin{(29) }%
\CSLRightInline{\href{https://doi.org/10.3390/metabo11040245}{Yamamoto,
M.; Shanmuganathan, M.; Hart, L.; Pai, N.; Britz-McKibbin, P.
\emph{Metabolites} \textbf{2021}, \emph{11} (4), 245}.}

\bibitem[\citeproctext]{ref-macedoSweatMetabolomeScreenPositive2017}
\CSLLeftMargin{(30) }%
\CSLRightInline{\href{https://doi.org/10.1021/acscentsci.7b00299}{Macedo,
A. N.; Mathiaparanam, S.; Brick, L.; Keenan, K.; Gonska, T.; Pedder, L.;
Hill, S.; Britz-McKibbin, P. \emph{ACS Central Science} \textbf{2017},
\emph{3} (8), 904--913}.}

\bibitem[\citeproctext]{ref-wishartHMDB50Human2022}
\CSLLeftMargin{(31) }%
\CSLRightInline{\href{https://doi.org/10.1093/nar/gkab1062}{Wishart, D.
S.; Guo, A.; Oler, E.; Wang, F.; Anjum, A.; Peters, H.; Dizon, R.;
Sayeeda, Z.; Tian, S.; Lee, B. L.; Berjanskii, M.; Mah, R.; Yamamoto,
M.; Jovel, J.; Torres-Calzada, C.; Hiebert-Giesbrecht, M.; Lui, V. W.;
Varshavi, D.; Varshavi, D.; Allen, D.; Arndt, D.; Khetarpal, N.;
Sivakumaran, A.; Harford, K.; Sanford, S.; Yee, K.; Cao, X.; Budinski,
Z.; Liigand, J.; Zhang, L.; Zheng, J.; Mandal, R.; Karu, N.; Dambrova,
M.; Schiöth, H. B.; Greiner, R.; Gautam, V. \emph{Nucleic Acids
Research} \textbf{2022}, \emph{50} (D1), D622--D631}.}

\bibitem[\citeproctext]{ref-wangCFMID40Web2022}
\CSLLeftMargin{(32) }%
\CSLRightInline{\href{https://doi.org/10.1093/nar/gkac383}{Wang, F.;
Allen, D.; Tian, S.; Oler, E.; Gautam, V.; Greiner, R.; Metz, T. O.;
Wishart, D. S. \emph{Nucleic Acids Research} \textbf{2022}, \emph{50}
(W1), W165--W174}.}

\bibitem[\citeproctext]{ref-dibattistaTemporalSignalPattern2017}
\CSLLeftMargin{(33) }%
\CSLRightInline{\href{https://doi.org/10.1021/acs.analchem.7b01727}{DiBattista,
A.; McIntosh, N.; Lamoureux, M.; Al-Dirbashi, O. Y.; Chakraborty, P.;
Britz-McKibbin, P. \emph{Analytical Chemistry} \textbf{2017}, \emph{89}
(15), 8112--8121}.}

\bibitem[\citeproctext]{ref-saoiMetabolicPerturbationsStep2019}
\CSLLeftMargin{(34) }%
\CSLRightInline{\href{https://doi.org/10.3390/metabo9070134}{Saoi, M.;
Li, A.; McGlory, C.; Stokes, T.; Allmen, M. T. von; Phillips, S. M.;
Britz-McKibbin, P. \emph{Metabolites} \textbf{2019}, \emph{9} (7),
134}.}

\bibitem[\citeproctext]{ref-pluskalMZmine2Modular2010}
\CSLLeftMargin{(35) }%
\CSLRightInline{\href{https://doi.org/10.1186/1471-2105-11-395}{Pluskal,
T.; Castillo, S.; Villar-Briones, A.; Orešič, M. \emph{BMC
Bioinformatics} \textbf{2010}, \emph{11} (1), 395}.}

\bibitem[\citeproctext]{ref-kassambaraPracticalGuidePrincipal2017}
\CSLLeftMargin{(36) }%
\CSLRightInline{KASSAMBARA, A. \emph{Practical {Guide} {To} {Principal}
{Component} {Methods} in {R}: {PCA}, {M}({CA}), {FAMD}, {MFA}, {HCPC},
factoextra}; STHDA, 2017.}

\bibitem[\citeproctext]{ref-MixOmicsPackageOmics}
\CSLLeftMargin{(37) }%
\CSLRightInline{\href{https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005752}{{mixOmics}:
{An} {R} package for `omics feature selection and multiple data
integration {\textbar} {PLOS} {Computational} {Biology}}.}

\bibitem[\citeproctext]{ref-alboukadelRstatixPipeFriendlyFramework2019}
\CSLLeftMargin{(38) }%
\CSLRightInline{\href{https://doi.org/10.32614/cran.package.rstatix}{Alboukadel,
K. \emph{CRAN: Contributed Packages} \textbf{2019}}.}

\bibitem[\citeproctext]{ref-pinheiroNlmeLinearNonlinear2011}
\CSLLeftMargin{(39) }%
\CSLRightInline{Pinheiro, J.
\href{https://cir.nii.ac.jp/crid/1370861707120203021}{Nlme: {Linear} and
nonlinear mixed effects models}, 2011.}

\bibitem[\citeproctext]{ref-grollVariableSelectionGeneralized2014}
\CSLLeftMargin{(40) }%
\CSLRightInline{\href{https://doi.org/10.1007/s11222-012-9359-z}{Groll,
A.; Tutz, G. \emph{Statistics and Computing} \textbf{2014}, \emph{24}
(2), 137--154}.}

\bibitem[\citeproctext]{ref-PROCOpensourcePackage}
\CSLLeftMargin{(41) }%
\CSLRightInline{\href{https://link.springer.com/article/10.1186/1471-2105-12-77}{{pROC}:
An open-source package for {R} and {S}+ to analyze and compare {ROC}
curves {\textbar} {BMC} {Bioinformatics}}.}

\bibitem[\citeproctext]{ref-villanuevaGgplot2ElegantGraphics2019}
\CSLLeftMargin{(42) }%
\CSLRightInline{\href{https://doi.org/10.1080/15366367.2019.1565254}{Villanueva,
R. A. M.; Chen, Z. J. \emph{Measurement: Interdisciplinary Research and
Perspectives} \textbf{2019}, \emph{17} (3), 160--167}.}

\bibitem[\citeproctext]{ref-weiPackageCorrplotVisualization2017}
\CSLLeftMargin{(43) }%
\CSLRightInline{Wei, T.; Simko, V. R package {``corrplot''}:
{Visualization} of a {Correlation} {Matrix} ({Version} 0.84), 2017.}

\bibitem[\citeproctext]{ref-PsychProceduresPersonality2017}
\CSLLeftMargin{(44) }%
\CSLRightInline{\href{https://CRAN.R-project.org/package=psych}{Psych:
{Procedures} for {Personality} and {Psychological} {Research}}, 2017.}

\bibitem[\citeproctext]{ref-klingelEPADHAHave2019}
\CSLLeftMargin{(45) }%
\CSLRightInline{\href{https://doi.org/10.1093/ajcn/nqz234}{Klingel, S.
L.; Metherel, A. H.; Irfan, M.; Rajna, A.; Chabowski, A.; Bazinet, R.
P.; Mutch, D. M. \emph{The American Journal of Clinical Nutrition}
\textbf{2019}, \emph{110} (6), 1502--1509}.}

\bibitem[\citeproctext]{ref-flockDeterminantsErythrocyteOmega32013}
\CSLLeftMargin{(46) }%
\CSLRightInline{\href{https://doi.org/10.1161/JAHA.113.000513}{Flock, M.
R.; Skulas‐Ray, A. C.; Harris, W. S.; Etherton, T. D.; Fleming, J. A.;
Kris‐Etherton, P. M. \emph{Journal of the American Heart Association}
\textbf{2013}, \emph{2} (6), e000513}.}

\bibitem[\citeproctext]{ref-potischmanBiologicMethodologicIssues2003}
\CSLLeftMargin{(47) }%
\CSLRightInline{\href{https://doi.org/10.1093/jn/133.3.875S}{Potischman,
N. \emph{The Journal of Nutrition} \textbf{2003}, \emph{133} (3),
875S--880S}.}

\bibitem[\citeproctext]{ref-lyLipidomicStudiesReveal2023}
\CSLLeftMargin{(48) }%
\CSLRightInline{\href{https://doi.org/10.1016/j.jlr.2023.100445}{Ly, R.;
MacIntyre, B. C.; Philips, S. M.; McGlory, C.; Mutch, D. M.;
Britz-McKibbin, P. \emph{Journal of Lipid Research} \textbf{2023},
\emph{64} (11)}.}

\bibitem[\citeproctext]{ref-dragstedValidationBiomarkersFood2018}
\CSLLeftMargin{(49) }%
\CSLRightInline{\href{https://doi.org/10.1186/s12263-018-0603-9}{Dragsted,
L. O.; Gao, Q.; Scalbert, A.; Vergères, G.; Kolehmainen, M.; Manach, C.;
Brennan, L.; Afman, L. A.; Wishart, D. S.; Andres Lacueva, C.;
Garcia-Aloy, M.; Verhagen, H.; Feskens, E. J. M.; Praticò, G.
\emph{Genes \& Nutrition} \textbf{2018}, \emph{13} (1), 14}.}

\bibitem[\citeproctext]{ref-vidgrenIncorporationN3Fatty1997}
\CSLLeftMargin{(50) }%
\CSLRightInline{\href{https://doi.org/10.1007/s11745-997-0089-x}{Vidgren,
H. M.; Ågren, J. J.; Schwab, U.; Rissanen, T.; Hänninen, O.; Uusitupa,
M. I. J. \emph{Lipids} \textbf{1997}, \emph{32} (7), 697--705}.}

\bibitem[\citeproctext]{ref-bouatraHumanUrineMetabolome2013}
\CSLLeftMargin{(51) }%
\CSLRightInline{\href{https://doi.org/10.1371/journal.pone.0073076}{Bouatra,
S.; Aziat, F.; Mandal, R.; Guo, A. C.; Wilson, M. R.; Knox, C.;
Bjorndahl, T. C.; Krishnamurthy, R.; Saleem, F.; Liu, P.; Dame, Z. T.;
Poelzer, J.; Huynh, J.; Yallou, F. S.; Psychogios, N.; Dong, E.;
Bogumil, R.; Roehring, C.; Wishart, D. S. \emph{PLOS ONE} \textbf{2013},
\emph{8} (9), e73076}.}

\bibitem[\citeproctext]{ref-zekiIntegrationGCMS2020}
\CSLLeftMargin{(52) }%
\CSLRightInline{\href{https://doi.org/10.1016/j.jpba.2020.113509}{Zeki,
Ö. C.; Eylem, C. C.; Reçber, T.; Kır, S.; Nemutlu, E. \emph{Journal of
Pharmaceutical and Biomedical Analysis} \textbf{2020}, \emph{190},
113509}.}

\bibitem[\citeproctext]{ref-dunnMetabolomicsCurrentAnalytical2005}
\CSLLeftMargin{(53) }%
\CSLRightInline{\href{https://doi.org/10.1016/j.trac.2004.11.021}{Dunn,
W. B.; Ellis, David. I. \emph{TrAC Trends in Analytical Chemistry}
\textbf{2005}, \emph{24} (4), 285--294}.}

\bibitem[\citeproctext]{ref-shanmuganathanCrossPlatformMetabolomicsComparison2021}
\CSLLeftMargin{(54) }%
\CSLRightInline{\href{https://doi.org/10.3389/fmolb.2021.676349}{Shanmuganathan,
M.; Sarfaraz, M. O.; Kroezen, Z.; Philbrick, H.; Poon, R.; Don-Wauchope,
A.; Puglia, M.; Wishart, D.; Britz-McKibbin, P. \emph{Frontiers in
Molecular Biosciences} \textbf{2021}, \emph{8}}.}

\end{CSLReferences}
\end{document}
